torchdata/dataloader2/dataloader2.py

```diff
@@ -95,6 +95,7 @@ def _pause(self) -> None:
         self.dataloader._pause()
         self.limit_counter = 0
 
+    @final
```

Comment:
nit: not a strong opinion, but I would guess if someone is inheriting from DataLoader2 to add things, they probably are intentionally rewriting some features

Reply:
I agree. Removed it with the latest commit